autonomy/fsm/scaffold/scaffold_skill.py

```diff
@@ -67,6 +68,11 @@
 from autonomy.fsm.scaffold.templates import COPYRIGHT_HEADER
 
 
+TO_LOCAL_REGISTRY_FLAG = "to_local_registry"
+VALORY_AUTHOR = "valory"
+VALORY_ABSTRACT_ROUND_SKILL_NAME = "abstract_round_abci"
```

Comment:
You can import the ABCI public id constant from here https://github.com/valory-xyz/open-autonomy/blob/be27dc89034c6ab28cdd04baecbc67d8757a6496/autonomy/constants.py#L60

Reply:
done